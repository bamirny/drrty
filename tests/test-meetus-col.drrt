def int main(){
	list[str] team;
	int i;
	list[str] job;
	list[str] img;
	
	createHTMLDocument("printbig.css");

	team = ["Dylan","Richard","Rania","Trinity"];


	job = ["Manager","Language Guru", "Tester", "System Architect"];

	

	makeHeader("Meet the DRRTY Team!");

	for (i = 0; i<length(team);i=i+1){
		createElement("div","column",team[i]);
		createElement("div","row",job[i]);
	}
	
	






return 0;

}