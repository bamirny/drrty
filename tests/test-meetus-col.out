<!DOCTYPE html><html><head><link rel="stylesheet" href="printbig.css"></head><body><h1>Meet the DRRTY Team!</h1>
<div class="column">Dylan</div>
<div class="row">Manager</div>
<div class="column">Richard</div>
<div class="row">Language Guru</div>
<div class="column">Rania</div>
<div class="row">Tester</div>
<div class="column">Trinity</div>
<div class="row">System Architect</div>
